WordPress/Classes/Models/AbstractPost.h

```diff
@@ -14,6 +14,14 @@ typedef NS_ENUM(NSUInteger, AbstractPostRemoteStatus) {
     AbstractPostRemoteStatusSync,       // Post uploaded
     AbstractPostRemoteStatusPushingMedia, // Push Media
     AbstractPostRemoteStatusAutoSaved,       // Post remote auto-saved
+
+    // All the previous states were deprecated in 24.7 and are no longer used
+    // by the app. To get the status of the uploads, use `PostCoordinator`.
+
+    /// The default state of the newly created local revision.
+    AbstractPostRemoteStatusLocalRevision,
+    /// The user saved the revision, and it needs to be uploaded to a server.
+    AbstractPostRemoteStatusSyncNeeded
```

Comment:
It's a better way than using `confirmedChangesHash` for two reasons:

- Once you confirm the save, the app "naturally" replaces the legacy `remoteStatus` with a new value `AbstractPostRemoteStatusSyncNeeded`
- Using integers is more efficient
- The DB migration in 24.8 will contain only removals – simpler